mergin/client.py

```diff
@@ -784,17 +787,19 @@ def add_user_permissions_to_project(self, project_path, usernames, permission_le
         :param usernames: list of usernames to be granted specified permission level
         :param permission_level: string (reader, writer, owner)
         """
-        if permission_level not in ["owner", "reader", "writer"]:
+        if permission_level not in ["owner", "reader", "writer", "editor"]:
             raise ClientError("Unsupported permission level")
 
```

Comment:
let's add an explicit check that if `permission_level == 'editor'` and server is old, we will throw a ClientError